src/Symfony/Bundle/TwigBundle/Resources/views/Exception/exception_full.html.twig

```diff
@@ -1,7 +1,136 @@
 {% extends '@Twig/layout.html.twig' %}
 
 {% block head %}
-    <link href="{{ absolute_url(asset('bundles/framework/css/exception.css')) }}" rel="stylesheet" type="text/css" media="all" />
+    <style>
+        .sf-reset .traces {
+            padding-bottom: 14px;
```

Comment:
we do same for svg icons inlining without controller (use twig include):
- https://github.com/symfony/symfony/blob/master/src/Symfony/Bundle/TwigBundle/Resources/views/layout.html.twig#L15
- https://github.com/symfony/symfony/blob/master/src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/results.html.twig#L38